src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+@RunWith(SpringRunner.class)
+@ActiveProfiles("functional")
+public class FindUsersByOrganisationTest extends FunctionalTestSuite {
+
+
+    @Test
+    public void find_users_by_active_organisation_with_showDeleted_False() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String organisationIdentifier = (String) response.get("organisationIdentifier");

```

Comment:
This creates an organisation which is pending, but the tests title mentions you are searching only in active organisation?

Reply:
This test first creates new PENDING organisation, updates organisation to ACTIVE and then it searches users for organisation created. So users will be retrieved only when organisation is active.
I think the title of test cases is ok.